Jenkinsfile

```
@@ -38,8 +38,20 @@ pipeline {
       }
       post{
       always {
-               recordIssues enabledForFailure: true, tool: checkStyle(pattern: 'target/checkstyle-result.xml')
-               recordIssues enabledForFailure: true, tool: pmdParser(pattern: 'target/pmd.xml')
+               recordIssues(
+                             enabledForFailure: true,
+                             blameDisabled: true,
+                             forensicsDisabled: true,
+                             qualityGates: [[threshold:0, type: 'TOTAL', unstable: false]],
+                             tool: checkStyle(pattern: 'target/checkstyle-result.xml')
+                            )
+               recordIssues(
+                             enabledForFailure: true,
+                             blameDisabled: true,
+                             forensicsDisabled: true,
+                             qualityGates: [[threshold:5, type: 'TOTAL', unstable: false]],
```

for no warnings in PMD, threshold should be set to 0
```suggestion
                             qualityGates: [[threshold:0, type: 'TOTAL', unstable: false]],
```

---

DOne